tools/pythonpkg/tests/arrow/test_parallel.py

```diff
@@ -43,26 +43,6 @@ def test_parallel_types_and_different_batches(self,duckdb_cursor):
             result = rel.filter("first_name=\'Jose\' and salary > 134708.82").aggregate('count(*)')
             assert (result.execute().fetchone()[0] == 4)
 
-    def test_parallel_types_and_different_batches(self,duckdb_cursor):
-        if not can_run:
-            return
-        duckdb_conn = duckdb.connect()
-        duckdb_conn.execute("PRAGMA threads=4")
-        duckdb_conn.execute("PRAGMA force_parallelism")
-
-        parquet_filename = os.path.join(os.path.dirname(os.path.realpath(__file__)),'data','userdata1.parquet')
-        cols = 'id, first_name, last_name, email, gender, ip_address, cc, country, birthdate, salary, title, comments'
-
-        userdata_parquet_table = pyarrow.parquet.read_table(parquet_filename)
-        for i in [7, 51, 99, 100, 101, 500, 1000, 2000]:
```

Comment:
why is this gone?

Reply:
it's the same from line 26-44, right?

I guess was duplicated by mistake before